gkeys/gkeysgpg/actions.py

```diff
@@ -0,0 +1,179 @@
+                    self.logger.info(_unicode(
+                        "Failed to find.........: %s in category: %s")
+                        % (args.category, args.nick))
+                args.category = self.config.get_key('verify-keyring')
+                args.nick = self.config.get_key('verify-nick')
+                self.logger.info(_unicode("Using config defaults..: %s %s")
+                    % (args.category, args.nick))
+                catdir = self._set_category(args.category)
+                return self.verify(args)
+
+        self.logger.debug(_unicode("ACTIONS: verify; catdir = %s") % catdir)
+        if args.statusfd:
+            self.config.defaults['gpg_defaults'] = [
+                '--display-charset', 'utf-8',
+                '--status-fd', args.statusfd]
+        self.config.defaults['gpg_defaults'].extend(["--trust-model", "always"])
+        self.logger.info("Verifying file...")
+        results = self.gpg.verify_text(key, data.encode('utf-8'), args.verify)
+        keyid = key.keyid[0]
+        (valid, trust) = results.verified
+        # TODO verify that the key it is signed with is listed as a current
+        # gpg key for that dev, not an old one still in the keyring.
+        # Add a setting to trigger allowing old gpg keys to validate against
+        if valid:
+            self.logger.info(_unicode("Verification succeeded.: %s")
+                % (args.verify))
+            self.logger.info(_unicode("Key info...............: %s <%s>, %s")
+                % ( key.name, key.nick, keyid))
+            self.logger.info(_unicode("    category, nick.....: %s %s")
+                % (args.category, args.nick))
+        else:
+            self.logger.info(
+                _unicode("Verification failed....: %s") % (args.verify))
+            self.logger.info(_unicode("Key info...............: %s <%s>, %s")
+                % ( key.name, key.nick, keyid))
+            found, args, new_msgs = self.keyhandler.autosearch_key(args, results)
+            if found:
+                return self.verify(args)
+        sys.stdout.write(results.output)
+        sys.stderr.write('\n'.join(results.stderr_out))
+        self.logger.debug("gpg stdout results: \n%s\n" %str(results.output))
+        self.logger.debug("gpg returncode: \n%s\n" %str(results.returncode))
+        self.logger.debug("gpg stderr results: \n%s\n" %str(results.stderr_out))
+        return (results.returncode, results)
+
+
+    def sign(self, args):
+        '''Sign a file'''
+        print("Made it to the --sign option :)")
+        gkeyargs = Args()
+        gkeyargs.filename = args.sign
+        gkeys = gkeyActions(self.config, self.output, self.logger)
+        return gkeys.sign(gkeyargs)
+
+
+    def _committer_search(self, data):
+        username = None
+        nick = None
+        for line in data:
+            self.logger.debug("_committer_search: line: %s" % line)
+            m = re.match("committer (.*) <(.*)@.*>", line)
```

Comment:
Can we replace regex?
Also, can you fix variable naming to improve readability ?